<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>papelera</class>
 <widget class="QDialog" name="papelera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Papelera</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>Cuenta</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fecha</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Asiento</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Concepto</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Debe</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Haber</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Documento</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Diario</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ejercicio</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="borrar_todo_pushButton">
       <property name="text">
        <string>&amp;Borrar todo</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/borrar.png</normaloff>:/iconos/graf/borrar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="recuperar_pushButton">
       <property name="text">
        <string>&amp;Recuperar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/reciclaje.png</normaloff>:/iconos/graf/reciclaje.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refrescar_pushButton">
       <property name="text">
        <string>R&amp;efrescar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/rayo.png</normaloff>:/iconos/graf/rayo.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="salirpushButton">
       <property name="text">
        <string>&amp;Salir</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/application-exit.png</normaloff>:/iconos/graf/application-exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>salirpushButton</sender>
   <signal>clicked()</signal>
   <receiver>papelera</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>694</x>
     <y>429</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
